mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/js/GEOR_custom.js

```diff
@@ -0,0 +1,80 @@
+Ext.namespace("GEOR");
+
+GEOR.custom = {
```

Comment:
I think that lots of options are not mentioned in this file. Needs more work IMO.